flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -200,34 +733,17 @@ def add_bzs_from_bca(
                 wl_df[bnd_ii + 1] = tide_ii
 
         # Determine bnd points from reference overland model
-        gdf_locs = self.sf_model.forcing["bzs"].vector.to_gdf()
-        gdf_locs.crs = self.sf_model.crs
+        gdf_locs = self._model.forcing["bzs"].vector.to_gdf()
+        gdf_locs.crs = self._model.crs
 
         # HydroMT function: set waterlevel forcing from time series
-        self.sf_model.set_forcing_1d(
+        self._model.set_forcing_1d(
             name="bzs", df_ts=wl_df, gdf_locs=gdf_locs, merge=False
         )
 
-    def get_wl_df_from_offshore_his_results(self) -> pd.DataFrame:
```

Comment:
this should be now in the HazardOutput object related to the offshore HazardModel